packages/flutter/lib/src/material/text_field.dart

```diff
@@ -317,6 +317,8 @@ class TextField extends StatefulWidget {
     this.canRequestFocus = true,
     this.spellCheckConfiguration,
     this.magnifierConfiguration,
+    this.painter,
```

Comment:
nit: would it be more self-explanatory if this is called "backgroundPainter"?

Reply:
Renamed. It might better to rename the variable in RenderEditable too, but that would be a breaking change I guess.